<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SkillDetailsWidget</class>
 <widget class="QWidget">
  <layout class="QVBoxLayout">
   <item>
    <widget class="QGroupBox">
     <layout class="QGridLayout" name="heroSkillGridLayout"/>
    </widget>
   </item>
   <item>
     <layout class="QHBoxLayout">
      <item>
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="skillPropertiesBox">
        <layout class="QGridLayout">
         <item row="0" column="0">
          <widget class="QLabel">
           <property name="minimumWidth">
            <number>48</number>
           </property>
           <property name="text">
            <string>Name:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="skillName"/>
         </item>
         <item row="0" column="2">
          <widget class="QLabel">
           <property name="minimumWidth">
            <number>48</number>
           </property>
           <property name="text">
            <string>AnimType:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="skillAnimType"/>
         </item>
         <item row="0" column="4">
          <widget class="QLabel">
           <property name="minimumWidth">
            <number>48</number>
           </property>
           <property name="text">
            <string>Level:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QLabel" name="skillLevel"/>
         </item>
         <item row="0" column="6">
          <widget class="QLabel">
           <property name="minimumWidth">
            <number>48</number>
           </property>
           <property name="text">
            <string>Cost:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="7">
          <widget class="QLabel" name="skillManaCost"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel">
           <property name="minimumWidth">
            <number>48</number>
           </property>
           <property name="text">
            <string>Desc.:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="3">
          <widget class="QLabel" name="skillDesc"/>
         </item>
         <item row="1" column="4">
          <widget class="QLabel">
           <property name="minimumWidth">
            <number>48</number>
           </property>
           <property name="text">
            <string>Source:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="5" colspan="3">
          <layout class="QHBoxLayout">
           <item>
            <widget class="QCheckBox" name="sourceAbilityCheckBox">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string>From ability</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="sourceMemCheckBox">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string>From memory</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="sourceInvCheckBox">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string>From inventory (scroll / rune)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="sourceItemCheckBox">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string>From equipment</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QLabel">
           <property name="minimumWidth">
            <number>48</number>
           </property>
           <property name="text">
            <string>Desc.:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" colspan="5">
          <layout class="QHBoxLayout">
           <item>
            <widget class="QCheckBox" name="misBaseAreaCheckBox">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string>Area damage (alternative hit chance calculation)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="misBaseNoBlockCheckBox">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string>can not be blocked</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="misBaseDotCheckBox">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string>Damage Over Time (Get-Hit is ineffective)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="misBaseLeadCheckBox">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string>Leads monsters to the player</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="misBaseShroudCheckBox">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string>Interacts with Shroud</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="misBaseArrowCheckBox">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string>Arrow Missile</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="3" column="1" colspan="5">
          <layout class="QHBoxLayout">
           <item>
            <widget class="QCheckBox" name="misAreaCheckBox">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string>Area damage (alternative hit chance calculation)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="misNoBlockCheckBox">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string>can not be blocked</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="misDotCheckBox">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string>Damage Over Time (Get-Hit is ineffective)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="misLeadCheckBox">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string>Leads monsters to the player</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="misShroudCheckBox">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string>Interacts with Shroud</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="misArrowCheckBox">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string>Arrow Missile</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </spacer>
      </item>
     </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QPushButton" name="resetButton">
       <property name="toolTip">
        <string>Set skills to zero</string>
       </property>
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="maxButton">
       <property name="toolTip">
        <string>Set skills to maximum</string>
       </property>
       <property name="text">
        <string>Max</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="submitButton">
       <property name="text">
        <string>Submit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
